airflow/providers/google/cloud/log/gcs_task_handler.py

```diff
@@ -123,16 +163,6 @@ def _read(self, ti, try_number, metadata=None):
             log += local_log
             return log, metadata
 
-    def gcs_read(self, remote_log_location):
-        """
-        Returns the log found at the remote_log_location.
-
-        :param remote_log_location: the log's location in remote storage
-        :type remote_log_location: str (path)
-        """
-        bkt, blob = self.parse_gcs_url(remote_log_location)
-        return self.hook.download(bkt, blob).decode('utf-8')
-
     def gcs_write(self, log, remote_log_location):
         """
         Writes the log to the remote_log_location. Fails silently if no hook
```

Comment:
'Fails silently if no hook'
there is no hook any longer

Reply:
Updated.